core/src/test/java/apoc/export/ImportAndLoadCoreSecurityTest.java

```diff
@@ -0,0 +1,263 @@
+    private static final String RELATIVE_URL_WITH_FILE_SLASH = "file:" + RELATIVE_URL;
+    private static final String RELATIVE_URL_WITH_FILE_DOUBLE_SLASH = "file:/" + RELATIVE_URL;
+    private static final String RELATIVE_URL_WITH_FILE_TRIPLE_SLASH = "file://" + RELATIVE_URL;
+
+
+    private final String apocProcedure;
+    private final String exportMethod;
+    private final String fileName;
+
+    public ImportAndLoadCoreSecurityTest(String method, String methodArguments, String fileName) {
+        this.apocProcedure = "CALL " + method + methodArguments;
+        this.exportMethod = method;
+        this.fileName = fileName;
+    }
+
+    @Parameterized.Parameters(name = "Procedure: {0}{1}, fileName: {2}")
+    public static Collection<String[]> data() {
+        // transform `Pair(<KEY>, <VALUE>)` to `Pair(apoc.import.<KEY> , <VALUE>)`
+        List<Pair<String, String>> importAndLoadProcedures = IMPORT_PROCEDURES
+                .map(e -> Pair.of( "apoc.import." + e.getLeft() , e.getRight() ))
+                .collect(Collectors.toList());
+
+        // transform `Pair(<KEY>, <VALUE>)` to `Pair(apoc.load.<KEY> , <VALUE>)`
+        List<Pair<String, String>> loadProcedures = LOAD_PROCEDURES
+                .map(e -> Pair.of( "apoc.load." + e.getLeft() , e.getRight() ))
+                .toList();
+
+        importAndLoadProcedures.addAll(loadProcedures);
+
+        return getParameterData(importAndLoadProcedures);
+    }
+
+    private static List<String[]> getParameterData(List<Pair<String, String>> importAndLoadProcedures) {
+        // from a stream of fileNames and a List of Pair<procName, procArgs>
+        // returns a List of String[]{ procName, procArgs, fileName }
+
+        Stream<String> fileNames = Stream.of(ABSOLUTE_URL,
+                ABSOLUTE_URL_WITH_FILE_PREFIX,
+                ABSOLUTE_URL_WITH_FILE_SLASH,
+                ABSOLUTE_URL_WITH_FILE_DOUBLE_SLASH,
+                ABSOLUTE_URL_WITH_FILE_TRIPLE_SLASH,
+                RELATIVE_URL,
+                RELATIVE_URL_WITH_FILE_SLASH,
+                RELATIVE_URL_WITH_FILE_DOUBLE_SLASH,
+                RELATIVE_URL_WITH_FILE_TRIPLE_SLASH);
+
+        return fileNames.flatMap(fileName -> importAndLoadProcedures
+                        .stream()
+                        .map(
+                            procPair -> new String[] { procPair.getLeft(), procPair.getRight(), fileName }
+                        )
+                ).toList();
+    }
+
+    @ClassRule
+    public static DbmsRule db = new ImpermanentDbmsRule()
+            .withSetting(GraphDatabaseSettings.load_csv_file_url_root, TEMP_FOLDER)
+            .withSetting(GraphDatabaseInternalSettings.cypher_ip_blocklist, List.of(new IPAddressString("127.168.0.0/8")));
+
+
+    @BeforeClass
+    public static void setUp() throws IOException {
+        TestUtil.registerProcedure(db,
+                ImportJson.class, Xml.class, ImportCsv.class, ExportGraphML.class,
```

Comment:
Do we not want arrow procedures here as well?

Reply:
Strange that the load graphml prcocedure is in the ExportGraphML class, maybe add a comment here as I was about to comment asking why an export procedure class was loaded into an import test 😅

Reply:
Yes sorry, I thought I also added arrow, added. 
And added a comment as well